src/Frontend/Core/Header/MetaCollection.php

```diff
@@ -10,6 +10,17 @@
     /** @var MetaLink[] */
     private $metaLinks = [];
 
+    public function __construct()
+    {
+        /* @remark Sumocoders staging websites should not be tracked */
```

Comment:
now that I think of it, the proper place for this is the __toString, because now you can still overwrite it, sorry ^^